sand/controllers/transform.py

```diff
@@ -0,0 +1,245 @@
+import json
+import re
+import sys
+import traceback
+from typing import Dict, List, Callable, Any, Union, Iterable, Tuple
+
+from flask import jsonify, request
+from flask.blueprints import Blueprint
+from RestrictedPython import compile_restricted_function, safe_globals
+from werkzeug.exceptions import BadRequest
+
+from sand.models.table import Link, Table, TableRow
+from sand.models.transform import Context, Tdata, TransformRequestPayload
+from gena.deserializer import get_dataclass_deserializer
+
+transform_bp = Blueprint("transform", "transform")
+deserializer = get_dataclass_deserializer(TransformRequestPayload, {})
+
+
+def filter_traceback_errors() -> str:
+    """Filters traceback errors, removes sensitive information
+
+        Args:
+
+        Returns:
+            Error String without the sensitive information.
+    """
+    (exc, value, tb) = sys.exc_info()
+    tb = tb.tb_next
+    return "".join(traceback.format_exception(exc, value, tb))
+
+
+def transform_map(transform_func: Callable[[Any, Context], Any], data: Iterable[Tuple[int, List, Context]],
```

Comment:
`Iterable[Tuple[int, List, Context]]`: in order to make this more useful, create a variable call ItemIndex = int, Item = Any, and use it in tuple[ItemIndex, Item, Context].